source/administration/hipchat-migration-guidelines.rst

```diff
@@ -0,0 +1,73 @@
+
+Guidelines for Success
+-------------------------------
++ Review the high level steps outlined below in this guide.
++ Review links to in-depth documentation to help you with more detailed instructions on configuration settings.
++ Customize your instance to meet your unique end-user and security requirements. 
++ Contact us at any time to assist you.
+
+Step 1:  Set up your Mattermost Instance
+-------------------------------
+#. Download the `latest version <https://about.mattermost.com/download/>`_ of Mattermost.
+#. `Deploy <https://docs.mattermost.com/guides/administrator.html#installing-mattermost)>`_ Mattermost in your environment using the configuration that meets your organization’s needs for performance and scalability.
+
+
+Step 2:  Export your data from HipChat 
+-------------------------------
+
+HipChat Server / HipChat Data Server:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Atlassian recommends upgrading to the latest version of HipChat Server or HipChat Data Center for access to the Group Export Dashboard.  You can view their instructions on exporting your data `here <https://www.atlassian.com/partnerships/slack/migration>`_.
+
+**Using the Group Export Dashboard:**
+
+#. Log in to your Hipchat Server (e.g. hipchat.yourcompany.com)
+#. Click on Server Admin > Export.
+#. Select the data to export.
+#. In the Password and Confirm Password fields, create a password to protect your archive files. (Store this password as it is not saved anywhere else)
+#. Click Export, and once the export is done you will receive an email with a link to download the file.
+
+**Using the Command Line Interface**
+
+#. Go to CLI.
+#. Enter hipchat export --export  -p your_password.
+#. Once the export is done you will receive an email with a link to download the file.
+
+More detailed instructions for the Command Line Interface can be found `here <https://confluence.atlassian.com/hipchatdc3/export-data-from-hipchat-data-center-913476832.html>`_.
+
+
+HipChat Cloud: 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. Log in at yourdomain.hipchat.com/admin/
+#. Click on the Data Export tab
+#. Select the data to export.
+#. In the Password and Confirm Password fields, create a password to protect your archive files. (Store this password as it is not saved anywhere else.)
+#. Click Export, and once the export has completed, the export will be available to you in the admin console.
+
+
+Step 3: Transform your data to the Mattermost import format  
+-------------------------------
+#. Format your HipChat data to JSONL format.  We offer a comprehensive `guide <https://docs.mattermost.com/deployment/bulk-loading.html#data-format>`_ to assist in the data formatting requirements for our bulk loading process.  
```

Comment:
Should we mention the ETL example here even though it's not an example for HipChat?